events/README.md

```diff
@@ -0,0 +1,54 @@
+It's hosted on [github.com/bi-foundation/live-feed-api](https://github.com/bi-foundation/live-feed-api)
+
+
+## Getting started
+To enable and configure the first layer Live Feed API in factomd there is a new property section in factomd.conf:
+```
+; ------------------------------------------------------------------------------
+; Configuration for live feed API
+; ------------------------------------------------------------------------------
+[LiveFeedAPI]
+EnableLiveFeedAPI                     = true
+EventReceiverProtocol                 = tcp
+EventReceiverAddress                  = 127.0.0.1
+EventReceiverPort                     = 8040
+OutputFormat                           = protobuf
+MuteReplayDuringStartup               = false
+ResendRegistrationsOnStateChange      = true
+ContentFilterMode                     = OnRegistration 
+```
+
+Here is an overview of these options:
+
+| Property                          | Description                                                                         | Values      |
+| --------------------------------- | ----------------------------------------------------------------------------------- | ----------- |
+|  EnableLiveFeedAPI                | Turn the Live Feed API on or off                                            | true &#124; false
+|  EventReceiverProtocol            | The network protocol that is used to send event messages over the network.     | tcp &#124; udp |
+|  EventReceiverAddress             | The receiver endpoint address.                                                | DNS name &#124; IP address |
+|  EventReceiverPort                | The receiver endpoint port.                                                  | port number |
+|  OutputFormat                      | The output format in which the event sent.                                      | protobuf &#124; json |
+|  MuteReplayDuringStartup          | At startup factomd can replay all the events that were stored since that last fastboot snapshot. Use this property to turn that on/off.   | true &#124; false |
+|  ResendRegistrationsOnStateChange | It’s possible to choose whether the chain and entry commit registrations should only be sent once, followed by state change events vs resending them for every state change. The first option reduces overhead & network traffic, but requires the implementer to track which state changes belong to which chain or entry.| true &#124; false |
+|  ContentFilterMode                | This option will determine whether the external ID’s and content will be included in the event stream. There are three level settings for this. Please note that the combination of ResendRegistrationsOnStateChange = true and ContentFilterMode=Always will resend all dataon every state change. The maximum content size per entry is only 10KB, however with a large number of transactions per second this may add up to an undesirable amount of data. | Always &#124; OnRegistration &#124; Never |
```

Comment:
README documentation calls this "Always" but it should be "SendAlways" instead, because that's what the actual parameter is implemented as. (Alternatively, keep the documentation as "Always" and update instances of "SendAlways" to "Always"... either way works. Just need to make sure they match.)

Reply:
I still had some changes for that which were not pushed yet. When you pull again ContentFilterMode will be called AllowContent. The combination filter and always/never can be confusing. (is it always filtering or always sending)